helm_deploy/hmpps-temporary-accommodation-ui/values.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 generic-service:
   nameOverride: hmpps-temporary-accommodation-ui
-  serviceAccountName: hmpps-community-accommodation-api-service-account
+  serviceAccountName: hmpps-temporary-accommodation-service-account
```

Comment:
Looks like there's a dangling fixup! commit to autosquash.

Where does this name come from? I guess it has to match [the name we define on the platform](https://github.com/ministryofjustice/cloud-platform-environments/pull/15718/files#diff-fd4ee85b307ffdb00a1125f208a7ef5038d1a07b4d059508daa344723335e1cdR65)?

Reply:
fixedup!

>Where does this name come from? I guess it has to match [the name we define on the platform](https://github.com/ministryofjustice/cloud-platform-environments/pull/15718/files#diff-fd4ee85b307ffdb00a1125f208a7ef5038d1a07b4d059508daa344723335e1cdR65)?

yep exactly (there's a link to the PR in the original commit message)